app/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_executable(shell-app main.cpp)
+add_executable(Small-World main.cpp Domain/Dice/Dice.cpp Domain/Dice/Dice.h Domain/Tokens/TokenType.h Domain/Tokens/TokenSubtype.h Domain/Tokens/Token.h)
```

Comment:
This will most likely conflict with #4 once it is merged. Suggested to checkout this and wait for the #4 to merge before applying your patch

Reply:
Agree. Will do it that way.